src/loop.jl

```diff
@@ -65,6 +65,17 @@ function interval_loop(factory_like, model_kwargs::Dict,
             if storage.enabled
                 model_kwargs["storage_e0"] = storage.sd_table.InitialStorage
             end
+            if demand_flexibility.enabled
+                try
+                    model_kwargs["init_shifted_demand"] = zeros(
+                        size(bus_demand_flex_amt_dn, 1)
+                    )
+                catch e
+                    model_kwargs["init_shifted_demand"] = zeros(
+                        size(bus_demand_flex_amt_up, 1)
+                    )
+                end
```

Comment:
Which exceptions are we expecting to be caught here?

Reply:
I added this in case a profile in one direction was not provided. However, this shouldn't be a problem based on the functionality added to `read_demand_flexibility` in `read.jl`. I can remove the `try`-`catch` check

Reply:
Done.